schemas/fingerprint-server-api.yaml

```diff
@@ -1703,3 +1728,23 @@ components:
       required:
         - code
         - message
+    VelocityIntervals:
+      type: object
+      additionalProperties: false
+      nullable: true
+      properties:
+        'intervals':
+          type: object
+          additionalProperties: false
+          nullable: true
+          properties:
+            '5m':
+              type: integer
+              example: 1
+            '1h':
+              type: integer
+              example: 1
+            '24h':
+              type: integer
+              example: 1
```

Comment:
Could you extract `intervals` to a separate type , e.g`Intervals`?
This will improve the generated code in our SDKs, as we will have named type for this.